src/processor/swmetricstransformprocessor/factory.go

```diff
@@ -122,9 +122,6 @@ func validateConfiguration(config *Config) error {
 			if op.Action == AddLabel && op.NewValue == "" {
 				return fmt.Errorf("operation %v: missing required field %q while %q is %v", i+1, NewValueFieldName, ActionFieldName, AddLabel)
 			}
-			if op.Action == FilterDataPoints && op.DataPointValue == 0 {
```

Comment:
this line is causing that we can't filter 0 value datapoint, which I think is legitimate use case.